content/en/docs/ops/common-problems/security-issues/index.md

```diff
@@ -408,3 +408,84 @@ second ensure that [keys and certificates are being delivered](#keys-and-certifi
 
 If everything appears to be working so far, the next step is to verify that the right [authentication policy](/docs/tasks/security/authentication/authn-policy/)
 is applied and the right destination rules are in place.
+
+### Auto mTLS
+
+If you suspect [auto mTLS](/docs/reference/glossary#auto-mtls) is not working
+as expected, please first read the
+[documentation](/docs/tasks/security/authentication/authn-policy/#auto-mutual-tls)
+and understand that [workload level](/docs/tasks/security/authentication/authn-policy/#enable-mutual-tls-per-workload) 
+peer authentication policies require corresponding destination rule configuration.
```

Comment:
I think this is to say auto mTLS is not supported on workload level and that's why it needs corresponding DR? could we make it clear for this unsupported case so that the user can get the point very quickly?

Reply:
Even better, part of istioctl :slightly_smiling_face: . I understand its not easy though

Reply:
done.